react/week3/todolist/src/App.js

```diff
@@ -0,0 +1,217 @@
+import './App.css';
+import React, { useEffect, useState } from "react";
+import DatePicker from "react-datepicker";
+import PropTypes from 'prop-types';
+
+import "react-datepicker/dist/react-datepicker.css";
+
+
+function CreateTodo(props) {
+    const { id, description, deadline } = props;
+    const [todoStatus, setTodoStatus] = useState(true);
+    const [updateStatus, setUpdateStatus] = useState(true);
+    const [todoDescription, setTodoDescription] = useState(description)
+    function checkboxStatus() {
+        setTodoStatus(!todoStatus)
+    }
+
+    function handleUpdateStatus() {
+        setUpdateStatus(!updateStatus)
+    }
+
+    function inputChange(e) {
+        setTodoDescription(prev => e.target.value)
+    }
+
+    return (
+        <div className="todo-item">
+            <div className="item-description">
+                <input type="checkbox" onClick={checkboxStatus}></input>
+                <li className="todo-li" style={{ 'textDecoration': todoStatus ? 'none' : 'line-through' }} >
```

Comment:
you should also check if the updateStatus is false since the text shouldn't be crossed out while editing